src/Sparrow/Utils/ZstdLib.cs

```diff
@@ -145,30 +175,25 @@ private static void AssertSuccess(UIntPtr v, CompressionDictionary dictionary)
 
         public class CompressContext : IDisposable
         {
-            private void* _sdtx;
-            public void* Streaming => _sdtx != null ? _sdtx : (_sdtx = CreateStreaming());
             private void* _cctx;
             public void* Compression => _cctx != null ? _cctx : (_cctx = CreateCompression());
             private void* _dctx;
             public void* Decompression => _dctx != null ? _dctx : (_dctx = CreateDecompression());
 
-            private void* CreateStreaming()
-            {
-                var sdtx = ZSTD_createDStream();
-                if (sdtx == null)
-                {
-                    throw new OutOfMemoryException("Unable to create compression context");
-                }
-                return sdtx;
-            }
-
             private void* CreateCompression()
             {
                 var cctx = ZSTD_createCCtx();
                 if (cctx == null)
                 {
                     throw new OutOfMemoryException("Unable to create compression context");
                 }
+
+                if (PlatformDetails.Is32Bits)
+                {
+                    var rc = ZSTD_CCtx_setParameter(cctx, ZSTD_cParameter.ZSTD_c_windowLog, 16);
```

Comment:
Comment explaining what the `16` means here

Reply:
done

Reply:
Is that relevant only for the compression? what about the decompression?